src/lib/utils.ts

```diff
@@ -44,6 +52,24 @@ export function getCleanedMetadata(
 		.sort();
 	const similarity = documentMatch?.similarity ?? 0;
 
+	const documentDate =
+		"DokDat" in metadata && Array.isArray(metadata.DokDat)
+			? metadata.DokDat[0]
+			: "";
+
+	const dateFromTitle =
+		type === "Hauptausschussprotokoll"
+			? title?.match(/\d{2}\.\d{2}\.\d{4}/)?.[0]
+			: "";
+
+	const formattedDate = parseDate(
+		String(documentDate ? documentDate : dateFromTitle),
```

Comment:
could be written as `documentDate ?? dateFromTitle` which avoids ternary operator